File: 30dayscoding/groupAnagrams.php
class Solution {

    /**
     * @param String[] $strs
     * @return String[][]
     */
    function groupAnagrams($strs) {
        $result=[];
        foreach($strs as $str){
            $aux=str_split($str);
            sort($aux);
            $temp=implode(' ',$aux).''; //join array elements on a string
            $result[$temp][]=$str;
        }
        return $result;
    }
}

File: ProgrammingSkills/GroupAnagrams.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        dictionary = dict()          
        for i in strs:
            sortedS = ''.join(sorted(s))
            dictionary[sortedS] = dictionary.get(sortedS, [])
            dictionary[sortedS] += [s]
        
        return dictionary.values()
